build.py

```
@@ -3,45 +3,146 @@
+        "parso",
+        "PIL",
+        "pydeck",
+        "bokeh",
+        "plotly",
+        "PyQt5",
+        "PySide2",
+        "sqlite3",
+        "distutils",
+        "setuptools",
+        "wheel",
+    }
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)